sonic-utilities-tests/show_ip_int_test.py

```diff
@@ -0,0 +1,156 @@
+    os.environ["PATH"] += os.pathsep + scripts_path
+    os.environ["UTILITIES_UNIT_TESTING"] = "2"
+    os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = "multi_asic"
+    yield
+    os.environ["UTILITIES_UNIT_TESTING"] = "0"
+    os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = ""
+
+
+def verify_output(output, expected_output):
+    lines = output.splitlines()
+    ignored_intfs = ['eth0', 'lo']
+    for intf in ignored_intfs:
+        # the output should have line to display the ip address of eth0 and lo
+        assert len([line for line in lines if line.startswith(intf)]) == 1
+
+    new_output = '\n'.join([line for line in lines if not any(i in line for i in ignored_intfs)])
+    print(new_output)
+    assert new_output == expected_output
+
+
+@pytest.mark.usefixtures('setup_teardown_single_asic')
+class TestShowIpInt(object):
+
+    def test_show_ip_intf_v4(self):
+        return_code, result = get_result_and_return_code(" ipintutil")
+        assert return_code == 0
+        verify_output(result, show_ipv4_intf_with_multple_ips)
+
+    def test_show_ip_intf_v6(self):
+        return_code, result = get_result_and_return_code(" ipintutil -a ipv6")
+
+        assert return_code == 0
+        verify_output(result, show_ipv6_intf_with_multiple_ips)
+
+    def test_show_intf_invalid_af_option(self):
+        return_code, result = get_result_and_return_code(" ipintutil -a ipv5")
+        assert return_code == 1
+        assert result == show_error_invalid_af
+
+
+@pytest.mark.usefixtures('setup_teardown_multi_asic')
+class TestMultiAsicShowIpInt(object):
+
+    def test_show_ip_intf_v4(self):
+        return_code, result = get_result_and_return_code("ipintutil")
+        assert return_code == 0
+        verify_output(result, show_multi_asic_ip_intf)
+
+    def test_show_ip_intf_v4_asic0(self):
+        return_code, result = get_result_and_return_code("ipintutil -n asic0")
+        assert return_code == 0
+        verify_output(result, show_multi_asic_ip_intf)
+
+    def test_show_ip_intf_v4_all(self):
+        return_code, result = get_result_and_return_code("ipintutil -d all")
+        assert return_code == 0
+        verify_output(result, show_multi_asic_ip_intf_all)
+
+    def test_show_ip_intf_v6(self):
+        return_code, result = get_result_and_return_code("ipintutil  -a ipv6")
+        assert return_code == 0
+        verify_output(result, show_multi_asic_ipv6_intf)
+
+    def test_show_ip_intf_v6_asic0(self):
+        return_code, result = get_result_and_return_code("ipintutil -a ipv6 -n asic0")
+        assert return_code == 0
+        verify_output(result, show_multi_asic_ipv6_intf)
+
+    def test_show_ip_intf_v6_all(self):
+        return_code, result = get_result_and_return_code("ipintutil -a ipv6 -d all")
+        assert return_code == 0
+        verify_output(result, show_multi_asic_ipv6_intf_all)
+
+    def test_show_intf_invalid_af_option(self):
+        return_code, result = get_result_and_return_code(" ipintutil -a ipv5")
```

Comment:
Delete space between `"` and `ipintutil`

Reply:
Fixed in latest commit